Snowdog_Components/components/Molecules/table/_table.scss

```diff
@@ -84,6 +84,25 @@
         }
     }
 
+    &--tfoot-border {
```

Comment:
I'm not sure about this name. I don't think it explains the purpose of these styles well. Is it for every table footer with border? I don't think so. only for the ones that have `<th scope="row"` because we need to set borders differently then. what do you think about naming it `--tfoot-scope-row`? :)